src/components/NavigationItem/index.tsx

```diff
@@ -31,7 +33,7 @@ const NavigationItem = ({
 
   return (
     <Link
-      to={`/learn/${slug}`}
```

Comment:
If you get rid of the /learn it breaks the site.
For example from the homepage click on the learn page. It returns a 404 not found.

Reply:
disregard... I think I had a cached version of the old site.